internal/pkg/crawl/worker.go

```diff
@@ -158,7 +157,7 @@ func (w *Worker) unsafeCapture(item *queue.Item) {
 func (w *Worker) Stop() {
 	w.doneSignal <- true
 	for w.state.status != completed {
-		runtime.Gosched()
```

Comment:
This led me to observe unusual CPU usage while waiting for workers to shut down.